### ISSUE TITLE
Temperature units don't change when the unit is changed

### ISSUE THREAD

**Issue by brianjmurrell** [sopelj/hass-ember-mug-component#50]
* Home Assistant Version (eg. 2023.11): 2024.1.6
* Ember Mug Component Version (eg. 0.8.0): 0.9.1
* Which device(s) are you using for Bluetooth (e.g. Bluetooth Dongle / ESPHome Proxy): Bluetooth Dongle

### Description

When I change the temperature unit for a device to _F_, the desired and present temperature still display in _C_.

### What I Did

Changed from _C_ to _F_ at 1, but the units at 2 and 3 in the below screenshot are still in _C_ as you can see:
![image](https://github.com/sopelj/hass-ember-mug-component/assets/559317/5a6a7d7b-c0ed-45e2-af11-8fcfaf34ef09)

**Comment by brianjmurrell** [comment]
I just noticed [this note](https://github.com/sopelj/hass-ember-mug-component#debugging:~:text=The%20%22Temperature%20Unit%22%20selected%20will%20be%20written%20to%20the%20mug%2C%20but%20does%20not%20change%20the%20value%20displayed%20in%20Home%20Assistant.) in the README.

I wonder then, what is the point of being able to change the unit in this integration?  That is, why even have that entity if it doesn't actually change anything?  Surely it's only going to lead to more tickets like this.

**Comment by sopelj** [comment]
Hi! 

The "Temperature Unit" entity controls the Mug's internal Temperature unit and not the unit Home Assistant uses to display the temperature as that is controlled by Home Assistant itself. It is there to allow people to change their Mug's preferences via Home Assitant in case they also use the native app for example.

The entities will show values based on your preferences in Home Assistant. So, you can either change that globally or you can also control it on a per-entity basis. To do so, simply click on the desired entity, click the gear and change the "unif of measure":

![image](https://github.com/sopelj/hass-ember-mug-component/assets/5285941/2205ff4f-1ac6-4785-9924-afffa49c382d)
![image](https://github.com/sopelj/hass-ember-mug-component/assets/5285941/d76fe6dd-ad6c-48b3-abd0-be6a5307f051)

**Comment by brianjmurrell** [comment]
Oh yes.  Nice.  I suppose once you know what you need to know though.  :-)

I didn't know you could configure such individual items.

TIL.  Thanks!

**Comment by sopelj** [comment]
True! I debated about renaming it "Internal Temperature Unit", but I worried both that it would be too long and that it would break existing installs. I'll add that to the doc though, thanks for pointing it out! And glad I could help at all.
